src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -226,6 +226,18 @@ public interface IDurableOrchestrationClient
         /// <returns>A task that completes when the terminate message is enqueued if necessary.</returns>
         Task TerminateAsync(string instanceId, string reason);
 
+        /// <summary>
+        /// Suspends a running orchestration instance.
+        /// </summary>
+        /// <returns>A task that completes when the suspend message is enqueued if necessary.</returns>
+        Task SuspendAsync(string instanceId, string reason);
+
+        /// <summary>
+        /// Resumes a suspended orchestration instance.
+        /// </summary>
+        /// <returns>A task that completes when the resume message is enqueued if necessary.</returns>
```

Comment:
Let's add parameter descriptions for `instanceId` and `reason`. This is important for this project since we build API documentation from these XML comments.

Reply:
Sure! Added